packages/prisma/schema.prisma

```diff
@@ -668,6 +675,7 @@ model Booking {
   // Ah, made a typo here. Should have been routedFromRoutingFormRe"s"ponse. Live with it :(
   routedFromRoutingFormReponse App_RoutingForms_FormResponse?
   assignmentReason             AssignmentReason[]
+  creationSource               CreationSource?
```

Comment:
We don't provide a default here to ensure we have all our bases covered. Any null value here for new users should tell us that we might have missed a route.